### PR TITLE
Update composer.json

### PR DESCRIPTION
**Pull request by jeffersongoncalves**
I'm having trouble with the installation of dependence by the composer.
The mistake is giving not find the cakephp-settings directory.

And I checked the cakephp-plugins.php file and it is with this line:

'Settings' => $baseDir . '/vendor/cakemanager/cakephp-settings/'

And cakemanager checked the folder and is the plugin with the Settings name.

### CODE REVIEW

**Comment by bobmulder** [comment]
Hi @simaostephanie, 

I don't really get why this issue should raise... This has never happened before. What version of CakePHP are you using?

By the way; the tests of the PR still fail... Can you fix those?

**Comment by jeffersongoncalves** [comment]
Hi @bobmulder,

I am using the version of CakePHP 3.2.13.

**Comment by bobmulder** [comment]
Probably that changed something to plugins, but I am not sure. I've not faced the issue yet...

**Comment by jeffersongoncalves** [comment]
It may be something related to the composer. I'm using version 1.2.0.

File: composer.json [hunk 1]
```diff
@@ -13,9 +13,6 @@
     "cakephp/orm": "~3.0",
     "phpunit/phpunit": " 4.6.*@dev"
   },
-  "extra": {
-    "installer-name": "Settings"
```
Comment (jeffersongoncalves) [inline composer.json:17]: It is changing the installation directory of the plugin via composer.

**Pull request closed without merging**